en/django_start_project/README.md

```diff
@@ -118,6 +118,35 @@ ALLOWED_HOSTS = ['127.0.0.1', '.pythonanywhere.com']
 
 > Also add `.amazonaws.com` to the `ALLOWED_HOSTS` if you are using cloud9
 
+> If you are hosting your project on `Glitch.com`, let us protect the Django secret key that needs to 
+> remain confidential (otherwise, anyone remixing your project could see it) :
+>   * First, we are going to create a random secret key. Open the Glitch terminal again, and type the following command :
+>     {% filename %}.env{% endfilename %}
+>     ```bash
+>     python -c 'from django.core.management.utils import get_random_secret_key; \
+>           print(get_random_secret_key())'
+>     ```
+>     This should display a long random string, perfect to use a secret key for your brand new Django web site.
+>     We will now paste this key into a `.env` file that Glitch will only show you if you are the owner of the web site.    
+>   
+>   * Create a file `.env` at the root of your project and add the following property in it :
```

Comment:
Putting a space before a colon (`:`) is AFAIK a French convention not usually followed in English.
```suggestion
>   * Create a file `.env` at the root of your project and add the following property in it:
```